app/src/main/cpp/skyline/services/socket/bsd/IClient.h

```diff
@@ -70,11 +82,22 @@ namespace skyline::service::socket {
          */
         Result Connect(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response);
 
+        Result GetPeerName(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response);
+
+        Result GetSockName(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response);
+
+        Result GetSockOpt(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response);
+
         /**
          * @brief Places a socket in a state in which it is listening for an incoming connection
          */
         Result Listen(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response);
 
+        /**
+         * @url https://switchbrew.org/wiki/Sockets_services#Fcntl
```

Comment:
`Performs a control operation on an open file descriptor`
Keep the url link, that is good.

Reply:
Don't bother with comments